data/xml/schema.rnc

```diff
@@ -7,7 +7,12 @@ url = element url {
 }
 fixed-case = element fixed-case { MarkupText }
 tex-math = element tex-math { text }
-MarkupText = (text | b | i | url | fixed-case | tex-math )+
+sup = element sup { MarkupText }
+a = element a {
+  attribute href { xsd:anyURI },
+  MarkupText
+}
```

Comment:
I'm not sure if we actually _want_ to support arbitrary links in abstracts etc.

Reply:
This makes sense. In fact, the one [paper](https://aclanthology.org/2022.emnlp-demos.41/) this impacts actually doesn't include the relevant bit of the abstract in the paper itself.